wiki/Client/Interface/Chat_console/Highlight/id.md

```diff
@@ -0,0 +1,21 @@
+---
+tags:
+  - mention
+  - hilight
```

Comment:
```suggestion
  - highlight
```

Reply:
I should've read the reply before committing. I will revert this along with the unresolved suggestions.